gulpfile.js

```diff
@@ -91,37 +91,47 @@ function writeServiceWorkerFile() {
     cacheId: 'polymerproject',
     staticFileGlobs: [
       `${rootDir}/1.0/index.html`,
+      `${rootDir}/1.0/shell.html`,
       `${rootDir}/1.0/about.html`,
       `${rootDir}/elements/**`,
       `${rootDir}/js/*.js`,
       `${rootDir}/css/*.css`,
       `${rootDir}/bower_components/**/webcomponents-lite.min.js`,
     ],
-    navigateFallback: '/1.0/index.html',
+    dynamicUrlToDependencies: {
+      '/1.0/shell.html': [
```

Comment:
I think this should be the actual URL the service worker will load over the network—`/1.0/shell`—rather than the file on disk (`/1.0/shell.html`). If we request `/1.0/shell.html` the server should just reply with a redirect. I'm having a lot of trouble figuring out what's going on from the network panel :/

Also, the dependencies only need to include the stuff that the shell actually includes—which is basically `shell.html` and `head-meta.html`.